tests/js/server/dump/dump-setup-common.inc

```diff
@@ -44,6 +44,12 @@ if (isEnterprise) {
   egm = require("@arangodb/enterprise-graph");
 }
 
+function getNoShards(defaultNo) {
```

Comment:
I am not against adding this function and using it in the tests.
But the devel version of this PR did not add such function, and I currently do not see an open devel PR that would add it.
So why is this function added only in 3.11? Do we need to add it in a separate devel PR?